templates/error/404.html.twig

```diff
@@ -1,5 +1,5 @@
 {% extends 'error/general_http_error.html.twig'  %}
-{% block title %}{{ "OpenEMR 404 Error"|xlt }}{% endblock %}
+{% block title %}{{ "OpenEMR 404 Error"|xl|json_encode }}{% endblock %}
```

Comment:
goodness, sorry for my prior comment on this. this is html, so you were right to use xlt here (ie. revert to what you had :) )

Reply:
Reverted this back.